README.md

```diff
@@ -104,41 +135,6 @@ BUT the values will be used from pipeline environment variables. Template variab
 >    1. Have `Packaging (Read)` permission.
 >    2. Be issued by a user with permission to the feed either directly or via a group. An easy way for this is to give `Contributor` permissions the `[{project_name}]\Contributors` group under the `Feed Settings -> Permissions` page. The page has the url format: `https://dev.azure.com/{organization}/{project}/_packaging?_a=settings&feed={feed-name}&view=permissions`.
 
-> [!NOTE]
```

Comment:
I thought it would be good to drop this from to reduce confusion and complexity given using V1 is discouraged now, any concerns?

Reply:
Great.